lib/mongo/server/connection_pool/connection_pool_populator.rb

```diff
@@ -24,12 +24,19 @@ class ConnectionPoolPopulator
 
       attr_reader :options
 
-      def initialize(pool)
+      # @param [ Server::ConnectionPool ] The connection pool.
+      # @param [ Hash ] options The options.
+      #
+      # @option options [ Logger ] :logger A custom logger to use.
+      def initialize(pool, options = {})
         @pool = pool
         @options = @pool.options
```

Comment:
This line can be deleted

Reply:
I seem to have missed this, what options did the populator use before?

Reply:
The populator includes Loggable, so that’s what the options were used for before. Now that the Background Thread includes Loggable, I think the “includes Loggable” line also can be deleted.

Reply:
Okay removed that too.